### ISSUE TITLE
Cannot find source file: src/Version.cpp when building

### ISSUE THREAD

**Issue by ProfessorKaos64** [plexinc/plex-media-player#310]
I caught that in the master tree, the hash mismatch on the tbz download was fixed, now I get this output below. That file does not exist in source. 

```
-- web-client version: 2d9a3a7
-- [download 0% complete]
...
-- [download 97% complete]
-- [download 98% complete]
-- [download 99% complete]
-- [download 100% complete]
-- clang-tidy not enabled, pass -DCMAKE_EXPORT_COMPILE_COMMANDS=on to cmake to enable it
-- Configuring done
CMake Error at src/shared/CMakeLists.txt:3 (add_library):
  Cannot find source file:

    /build/plex-media-player-1.0.6.229+git+bsos/Build/src/Version.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


CMake Error: CMake can not determine linker language for target: shared
-- Generating done
-- Build files have been written to: /build/plex-media-player-1.0.6.229+git+bsos/Build
debian/rules:7: recipe for target 'override_dh_auto_configure' failed
make[1]: *** [override_dh_auto_configure] Error 1
make[1]: Leaving directory '/build/plex-media-player-1.0.6.229+git+bsos'
debian/rules:4: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
E: Failed autobuilding of package
```

Related to the "spring cleaning" commits?

https://github.com/plexinc/plex-media-player/blob/b5546a9d341ff485cbcb7b5e9e974139071a8032/src/core/Version.cpp.in

**Comment by tru** [comment]
Edit src/shared/CMakeLists.txt and change the path to src/core/Version.cpp

I will push that fix tomorrow

**Comment by ProfessorKaos64** [comment]
I just saw that and was about to comment. Great news, thank you. I think after this, I will be set to have a test build for Jessie and SteamOS by the looks of it. I saw the has mismatch was fixed (the later issue I got) and I did correctly build ssl support for cmake 3.5.x. Appreciate the help and for bearing with me.

**Comment by ProfessorKaos64** [comment]
I see this was fixed in aa0dcda , closing.
